app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As required for the project, kindly ensure that your application asks for confirmation of email after signing up. You are supposed to have `:confirmable`  enabled in the user model (user.rb file).

Reply:
Thanks so much, @tresorsawasawa. I will effect that